cherrypy/_cplogging.py

```diff
@@ -216,7 +215,8 @@ def error(self, msg='', context='', severity=logging.INFO,
         if traceback:
             exc_info = _cperror._exc_info()
 
-        self.error_log.log(severity, ' '.join((self.time(), context, msg)), exc_info=exc_info)
+        self.error_log.log(severity, ' '.join(
```

Comment:
I think, it's nicer to split lines between arguments of `log()` method, not inside argument expressions.
Also please not that we follow 120-chars line restrictions (there's no 80-char terminals nowadays), so you don't really have to split it.